Modsenfy.DataAccessLayer/Data/DatabaseContext.cs

```diff
@@ -32,8 +35,37 @@ public DatabaseContext(DbContextOptions<DatabaseContext> options)
     public DbSet<Artist> Artists { get; set; }
 
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+>>>>>>> main
     {
-        var connectionString = "Server=.\\SQLEXPRESS;Initial Catalog=Modsenfy;Trusted_Connection=True;TrustServerCertificate=True";
+        Database.EnsureCreated();
+    }
+
+    public DbSet<Track> Tracks { get; set; }
+
+    public DbSet<Request> Requests { get; set; }
+
+    public DbSet<TrackArtists> TrackArtists { get; set; }
+
+    public DbSet<Album> Albums { get; set; }
+
+    public DbSet<Entities.Stream> Streams { get; set; }
+
+    public DbSet<RequestStatus> RequestStatuses { get; set; }
+
+    public DbSet<User> Users { get; set; }
+
+    public DbSet<UserInfo> UserInfos { get; set; }
+
+    public DbSet<Image> Images { get; set; }
+
+    public DbSet<ImageType> ImageTypes { get; set; }
+
+    public DbSet<Artist> Artists { get; set; }
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+    {
+        var connectionString = "Data Source=34.118.70.5,1433;Initial Catalog=modsenfydb;User ID=sqlserver;Password=password;";
```

Comment:
Move connectionString to appsettings.json and access it using injecting IConfiguration